assignment-1-sql/robert-border.sql

```diff
@@ -0,0 +1,157 @@
+-- Question 1
+-- Find the most recent completed order where the basket size was > $500.
+
+SELECT *
+FROM fact_order
+WHERE charge_amount > 500
+ORDER BY completed_at DESC
+LIMIT 1;
+
+-- Question 2
+-- What GMV have we generated, how many orders were made per week in the last 8 weeks in Canada? Order the results from the most recent week to the oldest.
+
+SELECT date_trunc('week', o.created_at) AS WEEK
+  ,count(distinct o.id) as num_orders
+  ,sum(o.charge_amount) as GMV
+FROM fact_order o
+JOIN zones z ON z.id = o.zone_id
+JOIN regions r ON r.id = z.region_id
+JOIN countries c ON c.id = r.country_id
+WHERE c.name = 'Canada'
```

Comment:
nit: would probably avoid the last join by just referencing the country_id directly. also the query would probably be less expensive if you used a where clause on the created_at date rather than limit.